### ISSUE TITLE
Button with Link styling

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#682]
In order to maintain accessibility, "links" that do not lead to a new page, ie open a modal, should be "buttons" using the html "< button >" tag.  Because this is important for accessibility, we should make it easy to do through the design system.

This ticket:
Create a "LinkButton" component which has all of the styling options of a link, but renders an 'button' tag and accepts button props.

**Comment by ehoops-cz** [comment]
Is this different than a minimal button?
